crates/astria-sequencer/src/bridge/bridge_lock_action.rs

```diff
@@ -26,6 +32,8 @@ use crate::{
     transaction::action_handler::ActionHandler,
 };
 
+pub(crate) const DEPOSIT_BYTE_LEN: u128 = std::mem::size_of::<Deposit>() as u128;
```

Comment:
Don't use the Rust in-memory representation of types if you want deterministic results. There are no guarantees on ordering of fields or alignment paddding. You can get different results depending on Rust compiler or even architecture.

You can use `#[repr(C)]` to get a defined layout (and hence deterministic size), but even that is tricky if somebody changes the representation of one of `Deposit`'s fields.

For `Deposit` in partciular I notice that it contains `destination_chain_address: String`, the memory size of which is is 24 bytes on a 64bit host (64 bit pointer, 64 bit capacity and length), with the actual memory used being unbounded.

Reply:
good point on the `destination_chain_address`, i'll change this to calculate the byte size at runtime (similar to sequence actions)